packages/perseus/src/widgets/graded-group/graded-group.tsx

```diff
@@ -118,6 +121,9 @@ export class GradedGroup
         answerBarState: "HIDDEN",
     };
 
+    rendererRef: Renderer | null | undefined;
+    hintRendererRef: Renderer | null | undefined;
```

Comment:
Can you use `React.createRef<T>() here instead? 

```suggestion
    rendererRef = React.createRef<Renderer>();
    hintRendererRef = React.createRef<Renderer>();
```

Then you can pass them to the `ref` prop directly, like this `ref={this.rendererRef}`.  

This does mean that usages will need to change to access the `current` value (eg. `this.rendereRef.current`). 

Example: https://github.com/Khan/perseus/blob/d6381f7737ff44f1ec3d91c4ab25ce51f47b35b9/packages/perseus/src/components/math-input.tsx#L435